### PR TITLE
Making RESTful API call - Tutorial Review Document

### PR DESCRIPTION
**Pull request by breezy-codes**
Document for the review of a tutorial on the Splashkit Site.

The tutorial **Making RESTful API calls** was reviewed and updated to ensure all code runs correctly and uses top level statements. The document covers all checks made and links to the PR made for this review.

### CODE REVIEW

File: docs/Splashkit/Tutorials/Reviews/making-RESTful-API-call.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# How to make a RESTful API call using SplashKit
+
+Date Reviewed: Brianna Laird
+Reviewed by: 9th August 2024
+
+## Description
+
+This tutorial covers how to create a JSON object and make a RESTful API call using SplashKit. The tutorial goes over usin GET resources and POST resources. The tutorial also covers how to handle the response from the API call.
```
Comment (sharvani28) [inline docs/Splashkit/Tutorials/Reviews/making-RESTful-API-call.md:8]: Correct the spelling of "using".

**Review by sharvani28 — APPROVED** [review]
Everything looks good! Just make sure to correct the spelling as mentioned below.

**Review by Srainyyyy — APPROVED** [review]
Everything looks good and the typos have been fixed

File: docs/Splashkit/Tutorials/Reviews/making-RESTful-API-call.md (hunk 1 above)
Comment (breezy-codes) [reply docs/Splashkit/Tutorials/Reviews/making-RESTful-API-call.md]: Fixed :)

**Review by omckeon — APPROVED** [review]

**Pull request merged**